File: Assignment_No46/File5.java
import java.util.*;
import LB.Matrix;


class File5
{
    public static void main(String[] args)
    {
        int iRow = 0, iCol = 0, iRet1 = 0, iRet2 = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows");
        iRow = sobj.nextInt();

        System.out.println("Enter the number of columns");
        iCol = sobj.nextInt();

        MyMatrix mobj = new MyMatrix(iRow,iCol);

        mobj.Accept();
        mobj.Display();

        mobj.SwapRows();
        System.out.println("\nAfter swap : ");
        mobj.Display();      
    }
}

class MyMatrix extends Matrix
{
    public MyMatrix(int A, int B)
    {
        super(A,B);
    }

    public void SwapRows()
    {
        int i=0;
        int iSize = Arr[0].length;
        int[] iTemp= new int[iSize]; 

        for(i=0; i<= (iSize) ; i++)
        {
           
            System.out.println("row "+ (1+i));
                iTemp =  Arr[i];
                Arr[i] =  Arr[iSize-1-(i)];
                Arr[iSize-1-(i)] = iTemp;
            
        }

        // iTemp = Arr[0];
        // Arr[0] = Arr[1];
        // Arr[1] = iTemp;
    }
}

File: Assignment_No19/File5.c
#include<stdio.h>
/*
1 2 3 4 
5 6 7 8 
9 10 11 12
*/

void Pattern(int iRows, int iCol)
{
    int i=0, j=0, iCnt=0;

    for(i=iRows; i>0; i--)
    {
        for(j=1; j<=iCol; j++)
        {
            iCnt++;
            printf("%d\t", iCnt);
        }
            printf("\n");
    }
}
int main()
{
    int iValue1=0 , iValue2=0;

    printf("Enter the number of rows and columns\n");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}